### ISSUE TITLE
VIT-7045 is missing an article

### ISSUE THREAD

**Issue by AngledLuffa** [UniversalDependencies/UD_Italian-VIT#11]
In the quotes, the original in the VIT constituency treebank is `"i tartassati"`, whereas the UD version eliminates the `i`:

```
# sent_id = VIT-7045
# text = Il sidecar di Totò ne "tartassati" e a completare questa piccola antologia di cinema su due e tre ruote non poteva mancare la bicicletta di "ladri di biciclette".
```

**Comment by msimi** [comment]
Yes, right.

— Maria

**Comment by AngledLuffa** [comment]
Fixed in #12.  Thank you!
